dvc/utils/__init__.py

```diff
@@ -398,6 +398,15 @@ def relpath(path, start=os.curdir):
     return os.path.relpath(path, start)
 
 
+def path_isin(child, parent):
```

Comment:
Move this to `dvc.utils.fs` please.

Reply:
I thought, `dvc.utils.fs` were mostly related to low-level stuffs like inodes. Moving ...

Reply:
@Suor, I have moved it to `dvc.utils.fs` and the tests as well.